webapp/src/js/services/db-sync.js

```diff
@@ -32,17 +25,19 @@ angular.module('inboxServices').factory('DBSync',
 
     var readOnlyFilter = function(doc) {
       // don't try to replicate read only docs back to the server
-      return READ_ONLY_TYPES.indexOf(doc.type) === -1 &&
-             READ_ONLY_IDS.indexOf(doc._id) === -1 &&
-             doc._id.indexOf(DDOC_PREFIX) !== 0;
+      return (
+        READ_ONLY_TYPES.indexOf(doc.type) === -1 &&
+        READ_ONLY_IDS.indexOf(doc._id) === -1 &&
+        doc._id.indexOf(DDOC_PREFIX) !== 0
+      );
```

Comment:
IMO this looks worse (uses two extra vertical lines to make scope clear that was clear already), but it's not the end of the world. I wonder if there is a flag to turn this off?

Reply:
Not sure. Its tagline is "opinionated code formatter" and documented flags are IMO minimal. I think I'd get used to it and opt for consistency.